app/views/application_records/show.haml

```diff
@@ -1,15 +1,16 @@
-%h1 Application Record
+%h1.h2="#{@record.position.name} Application Record"
```

Comment:
Would that be assuming that people working on our app didn't know about
bootstrap and what those h classes are for?

On Mon, Jul 18, 2016 at 1:53 PM, David Faulkenberry <
notifications@github.com> wrote:

> In app/views/application_records/show.haml
> <https://github.com/umts/jobapps/pull/261#discussion_r71195461>:
>
> > @@ -1,15 +1,16 @@
> > -%h1 Application Record
> > +%h1.h2="#{@record.position.name} Application Record"
>
> If it's everywhere, then a generic CSS rule on h1 is in order.
>
> If it's just here, then there two alternatives leap to mind which would
> communicate our intent a little better:
>
>    1. Wrap the page in some kind of div (.smaller-headers) and then write
>    CSS specific to that: css .smaller-headers h1{ font-size: ??px; }
>    2. Writing a more descriptive CSS class to apply directly to the
>    headings: css h1.we-want-it-to-be-a-little-smaller{ font-size: ??px; }
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/umts/jobapps/pull/261/files/23523fb8ad5746d3af683e9b22c1a4f0dd5ef33b#r71195461>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AJMtBMudIxmaGbXVwlpFzYHpwu5Tvxmlks5qW72egaJpZM4JO7Qc>
> .
>